cdc/sink/codec/canal_flat.go

```diff
@@ -143,8 +143,8 @@ func (c *canalFlatMessage) getMySQLType() map[string]string {
 }
 
 type tidbExtension struct {
-	CommitTs    uint64 `json:"commit-ts"`
-	WatermarkTs uint64 `json:"watermark-ts"`
+	CommitTs    uint64 `json:"commit-ts,omitempty"`
```

Comment:
Could you rename `"commit-ts"` to `commitTs` ? Since canal-json use camelCase style.

Reply:
Good catch!

Reply:
Addressed!